src/enzo/Make.mach.CCA-cluster

```diff
@@ -0,0 +1,109 @@
+#=======================================================================
+#
+# FILE:        Make.mach.ubuntu-hardy
+#
+# DESCRIPTION: Makefile settings for a machine running Ubuntu 8.04
+#
+# AUTHOR:      Rick Wagner (rick@ucsd.edu)
+#
+# DATE:        2008-09-16
+#
+# This configuration assumes that build-essentials, gfortran, 
+# OpenMPI and HDF5 have been installed using apt-get.
+#
+#=======================================================================
+
+MACH_TEXT  = Generic Ubuntu 8.04
+MACH_VALID = 1
+MACH_FILE  = Make.mach.ubuntu-hardy
+
+#-----------------------------------------------------------------------
+# Install paths (local variables)
+#-----------------------------------------------------------------------
+
+LOCAL_HDF5_INSTALL = /mnt/xfs1/home/xinyuli/hdf5
+
+LOCAL_GRACKLE_INSTALL  = /mnt/xfs1/home/emerick/local
+
```

Comment:
Hey that's me!

But anyway. This file maybe doesn't need to be included in this PR.

Reply:
removed